vignettes/version_policy.Rmd

```diff
@@ -0,0 +1,48 @@
+---
+title: "R version support"
+output: 
+  rmarkdown::html_vignette:
+    toc: true
+vignette: >
+  \usepackage[utf8]{inputenc}
+  %\VignetteIndexEntry{R version support}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+R evolves every year. Each update brings many bug fixes and performance 
+improvements, but also sometimes changes the default value of some parameters
+(such as switching from `stringsAsFactors = TRUE` to `FALSE` in R 4.0.0).
+
+While there is benefit in supporting older R versions, doing so adds 
+maintenance burden, especially in a large ecosystem like **easystats** that
+contains 10 packages that are seamlessly supposed to work together. Therefore,
+**easystats** is supported (meaning each package is thoroughly tested) on the
+**current R** version, the future one (**R-devel**) and the **previous four**
+versions. This is the same as the [version policy of `tidyverse`
+ecosystem](https://www.tidyverse.org/blog/2019/04/r-version-support/).
+
+Supposing R updates keep happening once a year in Spring, this table shows the 
+corresponding latest R version and the oldest supported version:
+
+| Year | Latest R version | Oldest supported R version |
+|------|------------------|----------------------------|
+| 2024 | 4.4              | 4.0                        |
+| 2025 | 4.5              | 4.1                        |
+| 2026 | 4.6              | 4.2                        |
+| 2027 | 4.7              | 4.3                        |
+| 2028 | 4.8              | 4.4                        |
```

Comment:
I'm not sure if there will be a version 4.8, but this table is hypothetical, I know. But then I would rather add an example that shows a major version change. E.g., what is the oldest version we support when R 5.0 or R 5.1 is released?

Reply:
Hmm, I don't think it would be a good idea to use R 5.0; makes it seem like we know something that even the R core doesn't know 😆 

Rather, we can illustrate the support behaviour when major R version changes using the jump from R 3.6 to R 4.0.

Reply:
True ;-)
But do we know that there ever will be a version 4.8? We didn't had 3.8 either.

Reply:
Minor versions chosen are known to change prior to releases. So nobody has hard expectations about them. 

A change in major version is a big deal for everyone.